yjit.c

```diff
@@ -491,6 +514,12 @@ rb_RB_TYPE_P(VALUE obj, enum ruby_value_type t)
     return RB_TYPE_P(obj, t);
 }
 
+VALUE
+rb_INT2FIX(long i)
+{
+    return INT2FIX(i);
+}
```

Comment:
Might be nice to have our own safe functiion for this 🤔

Reply:
Oh hey, I guess we already do. D'oh! I'll switch over to using it.